core/src/com/unciv/ui/newgamescreen/GameOptionsTable.kt

```diff
@@ -141,6 +143,14 @@ class GameOptionsTable(
             addCheckbox("Disable starting bias", gameParameters.noStartBias)
             { gameParameters.noStartBias = it }
 
+    private fun Table.addRandomPlayersCheckbox() =
+            addCheckbox("Random number of Players", gameParameters.noStartBias)
+            { gameParameters.randomNumberOfPlayers = it }
+
+    private fun Table.addRandomCityStatesCheckbox() =
+            addCheckbox("Random number of City-States", gameParameters.noStartBias)
```

Comment:
you probably meant `gameParameters.randomNumberOfPlayers` and `gameParameters.randomNumberOfCityStates` instead of `gameParameters.noStartBias`

Reply:
LOL

Reply:
I wonder why it works despite that.

Reply:
Oh, that is the starting value.

Reply:
Fixed.

Reply:
> I wonder why it works despite that.

because the second parameter of `addCheckbox()` will just be toggled. The whatever you add in the lambda it will be executed. What actually happened here is that whenever you clicked  the checkbox you toggled the `noStartBias` option as well. So if you want to make this cleaner you can just remove the lambda. (A couple months ago it was necessary to change the option in the lambda, but then someone changed this but didn't change all usages of the function so some usages still have the useless lambda)